src/components/ScholarshipAmountField.jsx

```diff
@@ -90,15 +92,29 @@ function ScholarshipAmountField(props) {
           );
         })}
       </RadioGroup>
+      <FormHelperText error>
+        {(typeError || minAmountError || maxAmountError) && helperText}
+      </FormHelperText>
     </FormControl>
   );
 }
 
 ScholarshipAmountField.propTypes = {
-  amountType: PropTypes.oneOf(Object.keys(AmountType)).isRequired, // supports default value of empty string
-  minAmount: PropTypes.number.isRequired,
-  maxAmount: PropTypes.number.isRequired,
+  amountType: PropTypes.oneOf(Object.values(AmountType)),
+  minAmount: PropTypes.string.isRequired,
+  maxAmount: PropTypes.string.isRequired,
```

Comment:
You could also leave these as numbers and just assume that `0` is not a valid min or max amount value (and hence show an empty text field).

Reply:
To show an empty text field I would have to do the `amount || ''` hack like before.

Reply:
Yeah that's probably right